### ISSUE TITLE
Elements on mid note positions skipped by Screen Reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31449]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When elements such as dynamics, lines, expression text, staff text, etc are moved to a mid note position (ie not on an exact beat or beat divison that has a note or rest) they either do not read at all with a screen reader or sometimes read in only one direction (often moving backwards) with an Alt arrow but are skipped when moving with a right alt arrow or just arrows.

See the attached file to use with these steps:
1.  Put focus on the first note of LH bar 2 and it announces "B flat 3 crotchet, bar 2 beat 1 articulation accent above start of pedal".
2. Press right arrow.  Notice it announces "rest quaver/eighth ..." and does not speak anything about the expression text sim.
3. Put focus again on Bar 2 beat 1 and press Alt right arrow. several times.  Notice it m ooves across and speaks the lower note of the chord, the articulation and then the pedal and then again jumps to the rest as if the expression text does not exist.
4. Place focus on the last beat of bar 2 and use left arrow to move along the bar, notice again the expression text is skipped over and not announced.
5. Place focus on last beat of bar 2 and use Alt left arrow to move backwards element by element, this time after the quaver rest you hear "expression sim. bar 2 beat 1.2893", or something like that, it was really hard to hear fast!

These sorts of things are occurring all the time now and elements are regularly disappearing for blind users.  We rely on transcriptions made by other people and also on OMR, both are introducing these elements not attached exactly to beats as they move them to look correct.  This file was created by one of the worlds most experienced Braille music transcribers who has used MuseScore for a long time and likely the issue came with the initial OMR.  My clue something was missing was when the pedal just stopped!

### Supporting files, videos and screenshots

[Mid note sim skipped - Ex 2b Grade 7 The Clifftop Citidal.zip](https://github.com/user-attachments/files/24147936/Mid.note.sim.skipped.-.Ex.2b.Grade.7.The.Clifftop.Citidal.zip)

### In which versions of MuseScore Studio is this issue present?

4.6.4

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Additional context

This has become a significant issue with more elements being able to be moved to a mid note position of small decimle increments in recent versions rather than staying firmly anchored to their associated note. When it is a small shift for visual purposes even when it does read it is not helpful to have all these decimal places read. 

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
